nextflow_schema.json

```diff
@@ -10,22 +10,26 @@
             "type": "object",
             "fa_icon": "fas fa-terminal",
             "description": "Define where the pipeline should find input data and save output data.",
-            "required": [
-                "input"
-            ],
             "properties": {
-                "input": {
+                "reads": {
                     "type": "string",
-                    "fa_icon": "fas fa-dna",
-                    "description": "Input FastQ files.",
-                    "help_text": "Use this to specify the location of your input FastQ files. For example:\n\n```bash\n--input 'path/to/data/sample_*_{1,2}.fastq'\n```\n\nPlease note the following requirements:\n\n1. The path must be enclosed in quotes\n2. The path must have at least one `*` wildcard character\n3. When using the pipeline with paired end data, the path must use `{1,2}` notation to specify read pairs.\n\nIf left unspecified, a default pattern is used: `data/*{1,2}.fastq.gz`"
+                    "default": "data/*_R{1,2}.fastq.gz",
```

Comment:
This is no longer the default

Reply:
I believe it is then best to remove a default there? 
It's still in the `nextflow.config` though. https://github.com/nf-core/methylseq/blob/master/nextflow.config

Reply:
Ah right, it's about to not be: https://github.com/nf-core/methylseq/pull/194/files#diff-ce7465a8e67b3b9c95696db1146ca7e6328f075e08a3d64062b87aa8608fc4eaL13-R13 (also, stop looking at `master` 😉 )

Yeah the problem with having a default is that it's not really compatible with a parameter being `required` - if nothing is entered it uses the default and passes validation. So need to remove the default from the schema and set to `null` in the config.